src/smart-components/request/request-table-helpers.js

```diff
@@ -5,17 +5,19 @@ import { Button } from '@patternfly/react-core';
 import ExpandableContent from './expandable-content';
 import { timeAgo }  from '../../helpers/shared/helpers';
 
-export const createInitialRows = data =>
-  data.reduce((acc, { id, requester, created_at, updated_at, state, decision, content }, key) => ([
+export const createInitialRows = data => {
+  return data.reduce((acc, { id, requester, created_at, updated_at, active_stage, total_stages, state, decision, content }, key) => ([
     ...acc, {
       id,
       isOpen: false,
+      state,
       cells: [ <Fragment key={ id }><Link to={ `/requests/detail/${id}` }>
-        <Button variant="link" > { id } </Button></Link></Fragment>, requester,
-      timeAgo(created_at), timeAgo(updated_at), state, decision ]
+        <Button variant="link"> { id } </Button></Link></Fragment>, requester,
+      timeAgo(created_at), timeAgo(updated_at), `${active_stage} of ${total_stages}`, decision ]
     }, {
       parent: key * 2,
-      cells: [{ title: <ExpandableContent id={ id } content={ content } /> }]
+      cells: [{ title: <ExpandableContent id={ id } content={ content } state={ state } active_stage={ active_stage }/> }]
     }
   ]), []);
+};
```

Comment:
why is this bracket `}` and the bracket `{` at line 8 added + the `return` statement added on line 9?

Reply:
I had other statements here so the curly braces and the return were necessary - but those statements were removed so the brackets/return are no longer necessary - I will remove them.